query-engine/connector-test-kit-rs/query-engine-tests/tests/queries/data_types/json.rs

```diff
@@ -55,7 +55,7 @@ mod json {
 
                 insta::assert_snapshot!(
                   res,
-                  @r###"{"data":{"findManyTestModel":[{"json":"{}"},{"json":"{\"a\":\"b\"}"},{"json":"1"},{"json":"\"hello\""},{"json":"[1,\"a\",{\"b\":true}]"}]}}"###
+                  @r###"{"data":{"findManyTestModel":[{"json":"{}"},{"json":"{\"a\":\"b\"}"},{"json":"1"},{"json":"1.5"},{"json":"\"hello\""},{"json":"[1,\"a\",{\"b\":true}]"}]}}"###
```

Comment:
These snapshots shouldn't have changed. What caused this?

Reply:
I've covered an additional case in the test, returning a float directly in a JSON field, it covers a new path in the SQLite implementation because it's returned as REAL instead of JSON there